content/features.md

```diff
@@ -0,0 +1,50 @@
++++
+title="Features"
+weight=1
+layout="standalone"
++++
+
+
+<div class='grid'>
+  
+{{< feature title="Bill-of-Materials" >}}
+Insights into the contents of the app image.
+{{</>}}
+
+{{< feature title="Build Cache" align="right" >}}
+Robust build caching is used to improve performance.
+{{</>}}
+
+{{< feature title="Composable" >}}
+Multiple buildpacks can be used to create an app image.
+{{</>}}
+
+{{< feature title="Fast" >}}
+Only re-builds and uploads layers when necessary.
```

Comment:
I wonder if people would read this and think "so what, docker does this with layers too". I'm not sure if there's an easy way to say "Only re-builds layers when necessary, not when something unrelated changes."

Reply:
Would this block be one that we can or should replace with "Reusability"?

Reply:
I think "Reusability" is more about reusing the buildpacks, not the layers. i could be wrong though

Reply:
Yeah, agreed. I was thinking about replacing this block with "Reusability" not so much because they were related but because we could probably improve the way we want to convey _speed_/_optimizations_ and in the meantime we have a need to fit the Reuse block in.

Reply:
Made this change on the latest commit.